src/shared/wallet/account-section.tsx

```diff
@@ -481,11 +506,13 @@ class AccountSection extends React.Component<Props, State> {
         </Menu.Item>
       </Menu>
     );
+
     return (
       <Dropdown.Button
         type="primary"
         overlay={claimMenu}
         onClick={this.onClaimClickHandle(token)}
+        className="claimButton"
```

Comment:
<img width="603" alt="Screen Shot 2019-07-10 at 10 25 09 AM" src="https://user-images.githubusercontent.com/1387285/60990328-05d85500-a2fd-11e9-82c1-257af08b29e7.png">

just use disabled property.

https://ant.design/components/dropdown/#header

Reply:
I did the same but @nmhung1210  asked me this So , I changed back to this " This will disable the dropdown menu too. A user might need to do claimAs and generate an auth message even they aren't claimable." let me know If I need to revert this. thanks

Reply:
I would suggest changing as below:

```suggestion
    <Dropdown.Button
        type="primary"
        overlay={claimMenu}
        className="claimButton"
        trigger={["click", "hover"]}
      >
        {
          // @ts-ignore
          <Button type="primary" style={{
            borderTopLeftRadius: 4,
            borderBottomLeftRadius: 4,
            boxShadow: "none"
          }} onClick={this.onClaimClickHandle(token)} disabled={!this.state.claimable}>
            {t("account.claim")}
          </Button>
        }
      </Dropdown.Button>
```

And the css for the claimButton (src/client/stylesheets/main.scss) as below:
```suggestion
.claimButton > .ant-btn:first-child {
  padding: 0 !important;
  border: 0 !important;
}
```